<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SIPIPE CC Licence Generator</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>861</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Client Info</string>
    </property>
    <widget class="QLabel" name="le_ClientName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name :</string>
     </property>
    </widget>
    <widget class="QLabel" name="le_ClientPhone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone :</string>
     </property>
    </widget>
    <widget class="QLabel" name="le_ClientEmail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Email :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_ClientName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_ClientPhone">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="te_ClientEmail">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>691</width>
       <height>16</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="b_UpdateClient">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>32</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="L_EndDate">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>End date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Grace time(days)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="t_GraceDuration">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>200</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_StartDate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="L_StartDate">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dt_EndDate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>200</y>
      <width>110</width>
      <height>21</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Products</string>
    </property>
    <widget class="QComboBox" name="CB_ProductIdPicker">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="b_GenerateLicense">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>530</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate licens for selected machines</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_BrowseClientSeed">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Import license seeds for client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_DeleteClient">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>221</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete selected client from database</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_Clients">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>301</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select client</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>270</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Features</string>
    </property>
    <widget class="QListWidget" name="li_Features">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>241</width>
       <height>181</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>270</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Client machines</string>
    </property>
    <widget class="QListWidget" name="li_ClientPCs">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select all</string>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="te_OutputPath">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse output path</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
